shared/api/network.go

```diff
@@ -38,6 +38,10 @@ type Network struct {
 
 	// API extension: network
 	Managed bool `json:"managed" yaml:"managed"`
+
+	// API extension: clustering
+	State string   `json:"state" yaml:"state"`
+	Nodes []string `json:"nodes" yaml:"nodes"`
```

Comment:
Should be Location instead of Nodes

Reply:
So since it's the second time you spell it singular I guess you really mean it :). Done.

Reply:
We seem to have been consistently using the plural form for lists, so Locations is actually better, sorry :)